src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -345,15 +345,23 @@ protected function describeContainerDefinition(Definition $definition, array $op
                     $argument = $argument->getValues()[0];
                 }
                 if ($argument instanceof Reference) {
-                    $argumentsInformation[] = sprintf('Service(%s)', (string) $argument);
+                    $argumentsInformation[] = sprintf('Service (%s)', (string) $argument);
```

Comment:
same comment as on the other PR: this should be reverted

Reply:
done